docs/support/troubleshooting.md

```diff
@@ -160,3 +160,16 @@ net.core.rmem_default=8388608 // only add if CycloneDDS is confgured
     ```bash
     echo $RMW_IMPLEMENTATION  // if Cyclone DDS is configured, this command will return "rmw_cyclonedds_cpp"
     ```
+
+### Multicast is disabled
+
+If you get the error message `selected interface "lo" is not multicast-capable: disabling multicast`, run the following command to allow multicast.
```

Comment:
If the error message example is generalized, then in theory the note is no longer required. For example:

```suggestion
If you get the error message `selected interface "*interface_name*" is not multicast-capable: disabling multicast`, run the following command to allow multicast.
```

Reply:
Thank you for your comment. How about using `{}` or `<>` instead of `**`? I think it's used more in common to represent a variable.

Reply:
How about this? 13d68cf00e58777f02c5f3543145a4eff93f5ce0